classes/DbWrapper.php

```diff
@@ -41,6 +41,7 @@ class DbWrapper
      */
     public static function update(string $table, array $data, string $where = '', int $limit = 0, bool $null_values = false, bool $use_cache = true, bool $add_prefix = true): bool
     {
+        // TODO: This should be reported in the things to fix
         $result = Db::getInstance()->update($table, $data, $where, $limit, $null_values, $use_cache, $add_prefix);
         self::validateDBQuerySuccess();
```

Comment:
can we develop the todo? What would you like to see here ?

Reply:
I meant to have this issue reported on SonarQube :smile: 

![image](https://github.com/user-attachments/assets/232c5396-9664-4ee3-8151-9f28e1472ab2)

There's nothing to do actually.